maven-release-plugin/src/it/projects/prepare/MRELEASE-667/pom.xml

```diff
@@ -37,7 +37,7 @@
           </dependency>
         </dependencies>
         <configuration>
-          <arguments>-Prelease,!mrelease-677 ${arguments}</arguments>
```

Comment:
maybe leave to see if removal do not have side effects?
this usage pattern  `<arguments>.... ${arguments}</arguments>` is used a lot

Reply:
It is one change - now arguments is pass directly to Invoker, and here property `${arguments}` is not resolved so is passed as is.
When we parsed CLI such not resolvable property was not used.

Reply:
do you mean leaving `<arguments>-Prelease,!mrelease-677 ${arguments}</arguments>` is now failing the build when `${arguments}` is not defined?
uhm this doesn't look right to me. at least should not fail.

Reply:
exactly ... if any property in `arguments` are not resolvable build will fail it. 
It is a cost ... parsing everything and duplicate code or pass value as is.
 
It worked in the same way with old forked executor.

Reply:
I know of projects that depend on ${arguments}, so better to fix the code instead of adjusting this IT.

Reply:
Ok, I will print warning about unresolved properties in `arguments` and strip its before pass to Invoker.

Can be a case that will be need pass to invoker something like `${.*}` ?